### ISSUE TITLE
Python Error

### ISSUE THREAD

**Issue by sgpearse** [NCAR/VAPOR#3063]
Import the NetCDF file /glade/p/cisl/vast/vapor/Bugs/TBD

Create the following Python script with input variables u, v, and w, and output variables `vort` and `vortUV` on the XxYxZ grid.

```
from vapor_utils import *
a,b,c=CurlFinDiff(u,v,w,20,20,20)
vort=Mag(a,b,c)
w0=np.zeros((14,120,120))
a2,b2,c2=CurlFinDiff(u,v,w0,20,20,20)
vortUV=Mag(a2,b2,c2)
```

**Comment by clyne** [comment]
Do u, v, and w0 all have the same shape? If not, the error thrown seems reasonable

**Comment by clyne** [comment]
Also, there is no link to the file

**Comment by sgpearse** [comment]
@clyne I updated the link to the file.  The shapes are different, but shouldn't the test be failing?  The error arises in vaporgui, but not after pressing the "Test" button in the python interface.

**Comment by clyne** [comment]
You are correct. There should be an error message reported during the "test" phase.
